agent/engine/dockerstate/dockerstate_test.go

```diff
@@ -203,6 +203,52 @@ func TestAddPendingEBSAttachment(t *testing.T) {
 
 }
 
+func TestAddPendingEBSAttachmentExclusion(t *testing.T) {
```

Comment:
nit: perhaps we can add a comment on what this will be testing for. a bit unclear on what we mean by exclusion here